app/assets/sass/patterns/_check-your-answers.scss

```diff
@@ -1,14 +1,75 @@
+
+// please adjust, depending on length of questions and answers and number of sections
+$cya-first-column-width: 30% !default;
```

Comment:
I just realised that using a SASS variable isn't as clever as I first thought. Because that means that you can only set it once per app!